lib/avo/licensing/request.rb

```diff
@@ -0,0 +1,18 @@
+module Avo
```

Comment:
I am seeing an `uninitialized constant Net::HTTP` when I start a rails server with rails 3.3.4 and avo 3.10.6. Do we need to add the following ?
 ```ruby
require "net/http"
```

When I add this to the top of `avo.rb` initialiser the problem is resolved.

Reply:
Yeah, I've been seeing that as well. We'll add it to the initial file as well.